static/app/views/dashboardsV2/issueWidgetCard.tsx

```diff
@@ -182,8 +192,16 @@ class IssueWidgetCard extends React.Component<Props> {
       <ContextWrapper>
         <ContextMenu>
           <Link to={issuesLocation}>
-            <StyledMenuItem>{t('Open in Issues')}</StyledMenuItem>
+            <StyledMenuItem key="open-issues">{t('Open in Issues')}</StyledMenuItem>
           </Link>
+          <StyledMenuItem
+            key="duplicate-widget"
+            title="Duplicate Widget"
```

Comment:
Do we need `title` here? The other place where you add a context menu doesn't have `title`

Reply:
yeah, there's a very convoluted reason why I have title here. Because I'm using the `disabled` flag on `StyledMenuItem`, I have to use the `onSelect` to define what happens when it's clicked as opposed to defining it in an `onClick` on this component. `MenuItem` renders a child component `MenuTarget` in this case and the `onSelect` defined here is actually applied as the `onClick` on the child component. So with RTL, when I want to simulate user click, I need to do it on the `MenuTarget` child component. The title gets passed down to that child component, so it's an easy way to retrieve it and click on it in the test. Open to cleaner suggestions lol

Reply:
https://github.com/getsentry/sentry/pull/30673/files#diff-59cf9c8682e57f518e9a531f674b5525672f26f7b8151ff4329cabf09b1aa1a2R175

Reply:
a89509648246c862aa2db11f617656fe7d519ae0